litgpt/data/prepare_slimpajama.py

```diff
@@ -11,6 +11,8 @@
 
 
 class SlimPajamaDataRecipe(DataChunkRecipe):
+    is_generator = True # fix an exception related to DataChunkRecipe
```

Comment:
It's ok to fix this, thanks. I think we would want to drop the comment though because it's redundant

Reply:
Yes, I removed the comment.